pyo3-ffi/src/impl_/mod.rs

```diff
@@ -0,0 +1,20 @@
+#[cfg(Py_GIL_DISABLED)]
+mod atomic_c_ulong {
+    pub(crate) struct GetAtomicCULong<const SIZE: usize>();
```

Comment:
Is the size in bits or in bytes? I would naively expect the latter, but have no opinion.

Reply:
Thanks, I was thinking it's easier to think in bits here but `size_of()` returns bytes, so this is currently bugged 👍

Renamed to `WIDTH` 👍